docs/schema/crxcavator.md

```diff
@@ -74,10 +71,31 @@ Placeholder representation of a single G Suite [user object](https://developers.
 | price | Extension price in webstore if applicable |
 | report_link | URL of full extension report on crxcavator.io
 
- ### Relationships
+### Relationships
 
-- GSuiteUsers install ChromeExternsions.
+- GSuiteUsers install ChromeExtensions.
 
     ```
     (GSuiteUser)-[INSTALLS]->(ChromeExtension)
     ```
+
+
+## ChromeExtensionPermission
+
+Individual permission entries from the extension manifest file. This information is returned from the manifest object in the [Report](https://crxcavator.io/apidocs#tag/report) API. These permissions are defined in the [Chrome documentation](https://developer.chrome.com/apps/declare_permissions#manifest).
+
+
+| Field | Description |
+|-------|--------------|
+| firstseen| Timestamp of when a sync job first discovered this node  |
+| lastupdated |  Timestamp of the last time the node was updated |
+| id | The permission name" |
+| name | The specific permission declared in the manifest
+
+### Relationships
+
+- ChromeExtension declares ChromeExtensionPermission.
+
+    ```
+    (ChromeExtension)-[DECLARES]->(ChromeExtensionPermission)
```

Comment:
Would like at least another set of eyes on this PR.

@SecPrez what do you think about representing permissions (https://developer.chrome.com/apps/declare_permissions#manifest) as nodes?  Would it be better if this was a field (like a list) on the ChromeExtension itself?

My thought is that ChromeExtension-to-ChromeExtensionPermission is many-to-1 relationship, so it could be unwieldy if someone were to expand all nodes from a ChromeExtensionPermission node.

Hm... but it **would** be very helpful to ask the question "what extensions ask for this super powerful permission?".

Sorry for the rambling reply; I'm fine with the way it is right now but I wanted to make sure @SecPrez saw it too.

Reply:
If it is a many-to-1 permission then it will need a node per permission. We should see how many nodes we are going to import. If the number is too high then only import the security interesting permissions vs all of them.

Reply:
2789 permission nodes during the test run, not a huge total. We could trim out the specific URI permissions, which would cut out most of those if this is a problem.

Reply:
That sounds reasonable. No objections here.